muselog/__init__.py

```diff
@@ -73,3 +73,11 @@ def setup_logging(root_log_level: Optional[str] = None,
 
         datadog_handler = DataDogUdpHandler(**opts)
         root_logger.addHandler(datadog_handler)
+
+    # Log to stdout if enabled
+    if "ENABLE_STDOUT_LOG" in os.environ and os.environ["ENABLE_STDOUT_LOG"] == "True":
+        stdout_handler = logging.StreamHandler()
```

Comment:
We already have stdout logging via 'console handler' I think this option should be to set stdout formatting.